main.py

```diff
@@ -38,61 +38,52 @@
 async def main(datatype, output_path, checksum, dryrun, search_id):
     def typematch(typeid):
         match typeid:
-            case 3:
-                return True
-            case 4:
-                return True
-            case 11:
-                return True
-            case 13:
-                return True
-            case 16:
+            case 3 | 4 | 11 | 13 | 16:

```

Comment:
This is brilliant!
Is there any possibility to replace with switch case?

Will you recommend the following solution? :
```py
def typematch(typeid:int):
    TypeID = [“3”, “4”, “11”, “13”, “16”]
    return TypeID.count(str(typeid))
```

Reply:
I like if-else and switch-case condition more. Your way is not comprehensible enough for me.